packages/block-editor/src/components/navigation-inspector/navigation-option.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
```

Comment:
Is there any way we can avoid having `core-data` as a dependency of the block editor package? Block editor should be usable independently from WP.

Could we perhaps add the title as an attribute of the nav block, and then use `getBlock` to access it instead?